app/src/main/java/com/duckduckgo/app/global/view/FireDialog.kt

```diff
@@ -33,6 +34,10 @@ class FireDialog(context: Context, private val clearPersonalDataAction: ClearPer
     init {
         val contentView = View.inflate(context, R.layout.sheet_fire_clear_data, null)
         setContentView(contentView)
+
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.O) {
+            window?.decorView?.systemUiVisibility = View.NOT_FOCUSABLE
```

Comment:
Not sure why this fixes the issue, but I would prefer to rely on themes / styles if possible. I did a quick look and found this article interesting https://medium.com/halcyon-mobile/implementing-googles-refreshed-modal-bottom-sheet-4e76cb5de65b 
I think modifying our `BottomSheetDialog` style and adding the following should work:
```
<item name="android:windowIsFloating">false</item>
<item name="android:statusBarColor">@android:color/transparent</item>
```